Aptoide/aptoide-client-v8
# [feature/ASV-1394] Valentines promotion

**What does this PR do?**

   Changes resources/styles for the new valentines promotion and also additional layout changes on the promotions fragment. Reference design can be found on Zeplin (under "Valentines promotion").

**Database changed?**

   No

**How should this be manually tested?**

  Use dev webservices, i.e. dev.ws75, to check and test promotions. You can check if the flow still works and if it follows the design guidelines referenced in Zeplin. 

  Note that the download progress layout is still messy since Christmas promotions (talk to @phillipsgoncalves 🥇).

**What are the relevant tickets?**

  [ASV-1394](https://aptoide.atlassian.net/browse/ASV-1394)


**Code Review Checklist**
